src/Columns/IColumnDummy.cpp

```diff
@@ -48,15 +49,14 @@ StringRef IColumnDummy::serializeValueIntoArena(size_t /*n*/, Arena & arena, cha
     return { res, 1 };
 }
 
-const char * IColumnDummy::deserializeAndInsertFromArena(const char * pos)
+void IColumnDummy::deserializeAndInsertFromArena(ReadBuffer & in)
 {
     ++s;
-    return pos + 1;
+    in.ignore(1);
 }
 
-const char * IColumnDummy::skipSerializedInArena(const char * pos) const
+void IColumnDummy::skipSerializedInArena(ReadBuffer &) const
 {
-    return pos;
```

Comment:
This looks odd. It seems it was incorrect before (and consequently now too), since it should skip once byte.